app/models/explainer.rb

```diff
@@ -117,7 +117,7 @@ def self.search_by_similarity(text, language, team_id, limit)
       context: context
     }
     response = Bot::Alegre.query_sync_with_params(params, "text")
-    results = response['result'].to_a.sort_by{ |result| result['_score'] }
+    results = response['result'].to_a.sort_by{ |result| [result['model'] != Bot::Alegre::ELASTICSEARCH_MODEL ? 1 : 0, result['_score']] }.reverse
```

Comment:
@melsawy , I have a refactoring suggestion: Implement a method in Alegre Bot v2 that receives a response from Alegre and then sorts the results, and then you can use this method in the two places, instead of repeating code... it will be also easier to unit-test and to document/explain using a comment.

Reply:
The format not same for Explainer & FactCheck
FactCheck sort is: `.sort_by{ |a| [a[1][:model] != Bot::Alegre::ELASTICSEARCH_MODEL ? 1 : 0, a[1][:score]] }.to_h.keys.reverse`
Explainer is: `.to_a.sort_by{ |result| [result['model'] != Bot::Alegre::ELASTICSEARCH_MODEL ? 1 : 0, result['_score']] }.reverse`

So I prefer to keep them independent.